CHANGELOG.md

```diff
@@ -132,6 +133,13 @@
 - `lokiexporter`: Log the first part of the http body on failed pushes to loki (#6946)
 - `resourcedetectionprocessor`: add the [consul](https://www.consul.io/) detector (#6382)
 - `awsemfexporter`: refactor cw_client logic into separate `cwlogs` package (#7072)
+- `lokiexporter`: add complete log record to body (#6619)
```

Comment:
Those changes do not look appropriate. They do not look like breaking changes. Please, revert them.

Reply:
hello @jpkrohling 
do you speak about:
- add the missing breaking change to the old 0.42
- merging the 2 enhancements blocs of the 0.42
- both
?

Reply:
The changes to the changelog should only pertain to the changes to this PR. If you think the changelog is wrong, it needs to be discussed and proposed on a separate PR.

Reply:
the other PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/7592